backend/pyproject.toml

```diff
@@ -10,8 +10,8 @@ readme = "README.md"
 python = "^3.10"
 fastapi = "^0.85.0"
 fastapi-users = "^10.1.5"
-surrealdb = {git = "https://github.com/surrealdb/surrealdb.py.git"}
 uvicorn = "^0.18.3"
+surrealdb = {git = "https://github.com/HakierGrzonzo/surrealdb.py"}
```

Comment:
leave here comment why you chose surrealdb from your repo ;P

Reply:
We are currently waiting for the upstream to merge some of my fixes, let's keep it like this so we can fix this client when it breaks.